internal/command/views/json/diagnostic.go

```diff
@@ -326,6 +326,11 @@ func NewDiagnostic(diag tfdiags.Diagnostic, sources map[string][]byte) *Diagnost
 							// in order to minimize the chance of giving away
 							// whatever was sensitive about it.
 							value.Statement = "has a sensitive value"
+						case val.HasMark(marks.Ephemeral):
+							if !includeEphemeral {
+								continue Traversals
+							}
+							value.Statement = "has an ephemeral value"
```

Comment:
was there a particular motivation behind hiding the fact that a value is sensitive when it's also ephemeral?

Reply:
I can add an extra clause for ephermal and sensitive values; the main concern here was not showing ephemeral values at all, they were displayed beforehand.

Reply:
I frankly now forgot where exactly this `Statement` gets displayed - I guess my main question was how did you come across this diagnostic in particular while working on `terraform test`? I just don't see it in any test being added.

I wasn't aware that ephemeral values were shown - that does sound like a bug - but again - how did you discover it? Shall we be adding a test for that somewhere?

Reply:
I can add an extra assertion. If you had a reference to an ephemeral variable in the condition or error message the value was shown.